decidim-core/spec/models/decidim/newsletter_spec.rb

```diff
@@ -20,12 +20,6 @@ module Decidim
         expect(subject).to be_valid
       end
 
-      it "is not valid without a body" do
```

Comment:
do we need to check for the content_block instead?

Reply:
We can't add a validation on the model because the command that creates newsletters first creates the newsletter and then links it to the content block model, and I can't find a way to do that at the same time... Both objects are created inside the same DB transaction, so we're safe.